tests/integration_tests/saas/test_hubspot_task.py

```diff
@@ -120,3 +123,82 @@ def test_saas_access_request_task(
             )
             > 0
     )
+
+
+@pytest.mark.integration_saas
+@pytest.mark.integration_hubspot
+def test_saas_erasure_request_task(
+        db,
+        policy,
+        erasure_policy_string_rewrite,
+        connection_config_hubspot,
+        dataset_config_hubspot,
+        hubspot_erasure_identity_email,
+        setup_teardown_erasure_hubspot_data,
+) -> None:
+    """Full erasure request based on the Hubspot SaaS config"""
+    privacy_request = PrivacyRequest(
+        id=f"test_saas_erasure_request_task_{random.randint(0, 1000)}"
+    )
+    identity_attribute = "email"
+    identity_value = hubspot_erasure_identity_email
```

Comment:
This var seems redundant since the only place we're using it is to assign a value in the dict below